bluebottle/projects/models.py

```diff
@@ -705,6 +708,14 @@ class Meta(BaseProject.Meta):
             ('api_add_project', 'Can add projects through the API'),
             ('api_change_project', 'Can change projects through the API'),
             ('api_delete_project', 'Can delete projects through the API'),
+            ('api_read_projectdocument', 'Can view project documents through the API'),
+            ('api_add_projectdocument', 'Can add project documents through the API'),
```

Comment:
Maybe we can't the default parent-permission model here because `ProjectDocument` should not be readable by anyone other than initiator (and perhaps reviewer). Not sure how it is solved now.